package.json

```diff
@@ -32,7 +32,16 @@
     "i18n-prepare-untranslated": "node ./scripts/prepareUntranslated.js && ./node_modules/.bin/rip json2pot 'app/i18n/extracted/**/*.json' -c id -o app/i18n/pot/decrediton.pot",
     "i18n-assemble-translated": "node ./scripts/assembleTranslated.js && ./node_modules/.bin/rip po2json 'app/i18n/po/*.po' -m 'app/i18n/extracted/**/*.json' -c id -o app/i18n/translations --indentation 2 --sort-by-id",
     "i18n-check-docs": "node ./scripts/docsTranslationStatus.js",
-    "i18n-check-dupes": "node ./scripts/checkDuplicateTranslationIds.js"
+    "i18n-check-dupes": "node ./scripts/checkDuplicateTranslationIds.js",
+    "prettify": "prettier --write 'app/**/*.{js,jsx,css,json}'",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.{jsx,jsx,css,json}": [
+      "yarn prettify",
+      "yarn lint-fix",
+      "git add"
```

Comment:
I don't think git should add with this script

Reply:
True, it's redundant in this case.